File: Java-Core/src/main/java/presto/python/Python3NativeCategoryBinding.java
package presto.python;

import presto.utils.CodeWriter;


public class Python3NativeCategoryBinding extends PythonNativeCategoryBinding {

	public Python3NativeCategoryBinding(PythonNativeCategoryBinding mapping) {
		super(mapping);
	}

	@Override
	public void toDialect(CodeWriter writer) {
		writer.append("Python3: ");
		super.toDialect(writer);
	}

}

File: Java-CodeServer/src/main/resources/js/prompto/python/Python3NativeCategoryBinding.js
var PythonNativeCategoryBinding = require("./PythonNativeCategoryBinding").PythonNativeCategoryBinding;

function Python3NativeCategoryBinding(binding) {
	PythonNativeCategoryBinding.call(this, binding.identifier, binding.module);
	return this;
}

Python3NativeCategoryBinding.prototype = Object.create(PythonNativeCategoryBinding.prototype);
Python3NativeCategoryBinding.prototype.constructor = Python3NativeCategoryBinding;

Python3NativeCategoryBinding.prototype.toDialect = function(writer) {
    writer.append("Python3: ");
    writer.append(this.identifier);
    if(this.module!=null)
        this.module.toDialect(writer);
};

exports.Python3NativeCategoryBinding = Python3NativeCategoryBinding;

